lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if ('max_team_size' in course_module.teams_configuration and
+            team.users.count() >= course_module.teams_configuration['max_team_size']):
```

Comment:
The inline style guide in PyCharm is complaining at me on this line, but https://www.python.org/dev/peps/pep-0008/ seems to indicate that this is an acceptable way to handle the conditional part of an if statement being too long for a single line. Any guidance?